pandas/tests/arrays/sparse/test_array.py

```diff
@@ -693,17 +691,13 @@ def test_getslice_tuple(self):
         dense = np.array([np.nan, 0, 3, 4, 0, 5, np.nan, np.nan, 0])
 
         sparse = SparseArray(dense)
-        res = sparse[
-            4:,
-        ]  # noqa: E231
-        exp = SparseArray(dense[4:,])  # noqa: E231
+        res = sparse[4:]
```

Comment:
since the test is called test_getslice_tuple

```suggestion
        res = sparse[(slice(4, None),)]
```

could be more appropriate?

Reply:
Ah, I missed the test name. So then it is probably indeed testing that passing a 1-len tuple is still working for a 1D array. Good catch!

Reply:
yes, you're right, thanks, I should've caught that